android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -68,13 +69,15 @@
   public double currentFrameTimeMs;
   public final UpdateContext updateContext;
   public Set<String> nativeProps = Collections.emptySet();
+  public Set<String> jsPropsHandledNatively = Collections.emptySet();
```

Comment:
Let's use `uiProps` for what used to be `nativeProps` and `nativeProps` for `jsPropsHandledNatively`